using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -2,68 +2,248 @@
+
+#### Event data example
+
+When all identifiers, blob and location hint are populated
+
+```json
+{
+    "mid": "midExample",
+    "advertisingidentifier": "advertisingIdExample",
+    "pushidentifier": "pushIdExample",
+    "blob": "blobExample",
+    "locationhint": "locationHintExample",
+    "visitoridslist": [{
+        "id_type": "type1",
+        "id_origin": "origin1",
+        "id": "id1",
+        "authentication_state": 1
+    },
+    {
+        "id_type": "type2",
+        "id_origin": "origin2",
+        "id": "id2",
+        "authentication_state": 2
+    }]
+}
+```
+
+#### Event data example
+
+The result for the appendToUrl request
+
+```json
+{
+    "updatedurl": "https://example.com/path?query=parameter1&other=parameter2&adobe_mc=TS%3D1563845864296%7CMCMID%3Dmidexample%7CMCORGID%3Dorgidexample%40AdobeOrg%7CMCAID%3Daidexample&adobe_aa_vid=videxample"
+}
+```
+
+#### Event data example
+
+The result for the getUrlVariables request
+
+```json
+{
+    "urlvariables": "adobe_mc=TS%3D1563845864296%7CMCMID%3Dmidexample%7CMCORGID%3Dorgidexample%40AdobeOrg%7CMCAID%3Daidexample&adobe_aa_vid=videxample"
+}
+```
 
 {% hint style="warning" %}
-The Analytics Custom Visitor ID \(VID\) is no longer included in the data payload. If needed, it may be retrieved from the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/174e9069bc1d3a521b59d52a066e9a7730f60ff5/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+
+The Analytics Custom Visitor ID \(VID\) is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+
 {% endhint %}
 
+### Configuration content request 
+
+The Identity Extension dispatches a new Configuration Content Request event when a privacy change is detected in the response received from the Experience Cloud ID Service. This event will be handled by the Core extension and the new privacy status will be broadcasted to other extensions.
+
+For more details about this event see the Mobile Core section.
+
+### Analytics content request
+
+The Identity Extension dispatches a new Analytics Content Request event when a push status preference change is detected, when the push identifier is initially set or when it is cleared from the SDK. This event will be handled by the Analytics extension and a new internal hit will be sent to the Adobe Analytics Server.
+
+For more details about this event see [Analytics Content Request](../../adobe-analytics/analytics-event-reference.md#analytics-content-request)
+
+
+
+## Shared state
+
+**EXTENSION\_NAME**: `com.adobe.module.identity`
+
+The Identity extension shared state is created in the following situations:
+
+* When the extension first loads and is initialized
```

Comment:
- Since the 3rd bullet is a complete sentence, to be consistent, all 3 bullet points require a period. 

- For the 2nd bullet, please make the following changes:
* Reword the bullet text as follows: "When handling a sync request, regardless of whether an actual sync occurs."
* Put the sentence "Occurs when..." on a new line under the bullet and add a period. 
* Add "the" before "Advertising Identifier" and "customer's custom identifiers" and add a period.

- Please add a period for the 3rd bullet.

Reply:
@rekrishn I made the changes except this point: 
- Put the sentence "Occurs when..." on a new line under the bullet and add a period.

The second sentence contains examples when the sync actually occurs, so it is related to the first sentence.